backuppc.json

```diff
@@ -3,8 +3,20 @@
   "release": "11.2-RELEASE",
   "artifact": "https://github.com/freenas/iocage-plugin-backuppc.git",
   "pkgs": [
+    "lang/perl5.26",
+    "sysutils/p5-BackupPC-XS",
+    "sysutils/p5-File-Listing",
+    "archivers/p5-Archive-Zip",
+    "textproc/p5-XML-RSS",
+    "www/p5-CGI",
+    "net/rsync",
+    "net/rsync-bpc",
     "sysutils/backuppc4",
-    "www/apache24"	  
+    "archivers/par2cmdline-tbb",
+    "net/samba48",
+    "www/apache24",
+    "www/mod_perl2",
```

Comment:
this is actually not needed since we use apaches cgi https://github.com/freenas/iocage-plugin-backuppc/blob/master/overlay/usr/local/etc/apache24/httpd.conf#L164